### DISCUSSION TITLE
💫 Yith Compare for WooCommerce ✨

### DISCUSSION THREAD

**Discussion by dxbuz** [inspireui/support#5724] [New Feature/Ideas Requests]
Hello!
Can you add Yith Compare plugin to Fluxstore?
This plugin will work same as a Yith Wishlist but only little difference, and I think this will not take a much time to integrate this plugin and this will be a great solution for many users to let their customers compare products.

This plugin has a free and premium version, I think it will be useful if you will integrate a free version and free version there is no many functions which is make integration easier

Free Version - https://wordpress.org/plugins/yith-woocommerce-compare/

Paid Version - https://yithemes.com/themes/plugins/yith-woocommerce-compare/

**Comment by Yan-Aleman** [comment]
The "electro" theme uses this plugin.  Indeed, it would be nice to have integration with the free version of this plugin.
